TM1py/Services/CellService.py

```diff
@@ -477,25 +537,15 @@ def extract_cellset_raw(
         elif 'Ordinal' not in cell_properties:
             cell_properties.append('Ordinal')
 
-        if not elem_properties:
-            elem_properties = ['UniqueName']
-        elif 'UniqueName' not in elem_properties:
-            elem_properties.append('UniqueName')
-
-        if not member_properties:
```

Comment:
Unfortunately we can't remove this code. Still needed to cater for potential calculated Members

Reply:
Hi Marius,
Yes, UniqueName is needed for the execute_mdx / extract_cellset / build_content_from_cells functions, but it might not needed for other uses so I did not want to force returning "UniqueName", except in those functions. The call from extract_cellset to extract_cellset_raw forces those properties (and only those properties) on element and members.

Reply:
that's a fair point. It does blow up the response size, so we should be careful with what we are querying from TM1, but I can't think of a use case where we would not want to cater for potential calculated members. Do you see one?

If there is no actual use case to call this function without asking for the member-unique-names I would prefer to play it safe and always query them.

Reply:
Hi Marius,
The member UniqueName and element UniqueName will both be queried for execute_mdx / extract_cellset / build_content_from_cells. 

My purpose does not need UniqueName so I was hoping to allow them to not be included when extracting raw.

Reply:
OK